src/dropin-model.js

```diff
@@ -139,6 +144,12 @@ DropinModel.prototype._shouldEmitRequestableEvent = function (options) {
   var requestableStateHasNotChanged = this.isPaymentMethodRequestable() === options.isRequestable;
   var typeHasNotChanged = options.type === this._paymentMethodRequestableType;
 
+  if (!this._setupComplete) {
+    // don't emit event until after Drop-in is fully set up
+    // fixes issues with lazy loading of imports where this event
```

Comment:
this comment seems incomplete. Maybe something either
`fixes issues with lazy loading of imports` OR `fixes issues with lazy loading of imports where this event shouldn't be emitted yet`